### DISCUSSION TITLE
How to modify the page " You have not been authorized "

### DISCUSSION THREAD

**Discussion by syed-shirazi** [kbeflo/evilportal-templates#52] [Q&A]
Hi everyone, can someone explain me please, once we have clicked on the button login on a evil portal, we are redirected to a page, it's written " you have not been authorized " .
I would like to know if we can modify it, and customize it.
Thanks

**Comment by NicoAgusAlma** [comment]
Still waiting for this answer. I can also see that evil portals and all about wifi pineaple has been aging without much updates.. it's a shame..

**Comment by kleo** [comment]
https://github.com/kleo/evilportals/discussions/41#discussion-3781459

Also my MK VI is not working anymore.
